### DISCUSSION TITLE
Github API v3 : retrieve all repos clone links in a given org

### DISCUSSION THREAD

**Discussion by mhow2** [community/community#21877] [New to GitHub]
<p>I’m trying to find a proper way to retrieve every repository (clone) link within an org. For example <a href="https://github.com/RocketChat" rel="nofollow noopener">https://github.com/RocketChat</a></p>
<p>From the <a href="https://developer.github.com/v3/projects/#list-organization-projects" rel="nofollow noopener">docs</a>, I understand I have to request : <a href="https://api.github.com/orgs/RocketChat/projects/" rel="nofollow noopener">https://api.github.com/orgs/RocketChat/projects/</a> and get the repo link field from there. But I only get :</p>
<pre><code>{
  "message": "If you would like to help us test the Projects API during its preview period, you must specify a custom media type in the 'Accept' header. Please see the docs for full details.",
  "documentation_url": "https://developer.github.com/v3"
}
</code></pre>
<p>What do I miss ?</p>

**Comment by seveas** [comment]
<p>According to the docs at <a href="https://developer.github.com/v3/projects/" rel="nofollow noopener">https://developer.github.com/v3/projects/</a>, this api is in preview, so you must specify a special Accept header to acknowledge that this is a preview that may change at any time.</p>

**Comment by mhow2** [comment, marked as answer]
<p>Answering my own question : the answer <a href="https://developer.github.com/v3/repos/#list-organization-repositories" rel="nofollow noopener">is in the doc</a> and is pretty straight-forward by broswing <a href="https://api.github.com/orgs/RocketChat" rel="nofollow noopener">https://api.github.com/orgs/RocketChat</a> , and look at the <strong>repos_url</strong></p>

**Comment by mhow2** [comment]
<p>Thank you for taking the time to answer. I posted a solution for my initial need.</p>
